coclico/mpla0/mpla0.py

```diff
@@ -17,9 +16,10 @@ class MPLA0(Metric):
     reference
 
     - metric_intrinsic:
-        for each input file, generate a tif file with one layer by class in the class_weight dictionary
-        for each class, the corresponding layer contains a kind of 2d occupancy map for the class (ie. if any point
-        of this class belongs to the pixel, the pixel has a value of 1, the value is 0 everywhere else)
+        for each input file, generate a tif file with one layer by class in the configuration file in
```

Comment:
suggestion
        for each input file, generate a tif file with one layer by class that has a weight in the configuration file
        for each class, the corresponding layer contains a kind of 2d occupancythe corresponding layer contains a kind of 2d occupancy

Reply:
OK !